Source/BasicMathFunctions/muriscv_nn_elementwise_mul_s8.c

```diff
@@ -114,44 +114,44 @@ muriscv_nn_status muriscv_nn_elementwise_mul_s8(const int8_t *input_1_vect,
     {
         /* Load vec 1 */
         int32_t packed = muriscv_nn_read_q7x4_ia(&input_1_vect);
-        int32_t a1 = __rv_sunpkd810(packed);
-        a1 = __rv_add16(a1, offset_1_packed);
-        int32_t b1 = __rv_sunpkd832(packed);
-        b1 = __rv_add16(b1, offset_1_packed);
+        int32_t a1 = __rv__sunpkd810(packed);
+        a1 = __rv__add16(a1, offset_1_packed);
+        int32_t b1 = __rv__sunpkd832(packed);
+        b1 = __rv__add16(b1, offset_1_packed);
 
         /* Load vec 2 */
         packed = muriscv_nn_read_q7x4_ia(&input_2_vect);
-        int32_t a2 = __rv_sunpkd810(packed);
```

Comment:
It that simply a change from `__rv_` to `__rv__` (number of underscores)?

Reply:
Oh s**t, I pushed some wrong changes. Will revert later…
Some context: I switched to another gcc based on RVP 0.9.6 where the prefixes have been different. And because of that I found out, that the MAX/MIN macros aren’t used everywhere.